src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -79,10 +89,9 @@ def send_historical_data_to_seer(alert_rule: AlertRule, project: Project) -> Bas
         or alert_rule.organization is None
     ):
         # this won't happen because we've already gone through the serializer, but mypy insists
-        base_error_response.reason = (
+        raise ValidationError(
             "Cannot create alert_rule - missing expected configuration for a dynamic alert."
```

Comment:
oops - missed this too
nit: lets not tie this msg explicitly to the alert_rule either, just the second half is good 👍